### ISSUE TITLE
`kctrl`: package repository kick doesn't support positional args while using package command tree

### ISSUE THREAD

**Issue by praveenrewar** [carvel-dev/kapp-controller#1042]
**What steps did you take:**
`tanzu package repository kick -h`

**What happened:**
```
Trigger reconciliation for repository

Usage:
  package repository kick [flags]

Flags:
  -h, --help                           help for kick
  -n, --namespace string               Specified namespace ($KCTRL_NAMESPACE or default from kubeconfig)
  -r, --repository string              Set repository name (required)
      --tty                            Force TTY-like output (default true)
      --wait                           Wait for reconciliation to complete (default true)
      --wait-check-interval duration   Amount of time to sleep between checks while waiting (default 1s)
      --wait-timeout duration          Maximum amount of time to wait in wait phase (default 5m0s)

Global Flags:
      --column strings              Filter to show only given columns
      --debug                       Include debug output
      --kube-api-burst int          Set Kubernetes API client burst limit (default 1000)
      --kube-api-qps float32        Set Kubernetes API client QPS limit (default 1000)
      --kubeconfig string           Path to the kubeconfig file ($TANZU_KUBECONFIG)
      --kubeconfig-context string   Kubeconfig context override ($TANZU_KUBECONFIG_CONTEXT)
      --kubeconfig-yaml string      Kubeconfig contents as YAML ($TANZU_KUBECONFIG_YAML)
  -y, --yes                         Assume yes for any prompt
```

**What did you expect:**
Repository name should be accepted using positional arguments.

**Environment:**

- kapp Controller version (execute `kubectl get deployment -n kapp-controller kapp-controller -o yaml` and the annotation is `kbld.k14s.io/images`):
- Kubernetes version (use `kubectl version`)

---
Vote on this request

This is an invitation to the community to vote on issues, to help us prioritize our backlog. Use the "smiley face" up to the right of this comment to vote.

👍 "I would like to see this addressed as soon as possible"
👎 "There are other more important things to focus on right now"

We are also happy to receive and review Pull Requests if you want to help working on this issue.

**Comment by praveenrewar** [comment]
This issue is still present as the value from the positional args is not getting used.

**Comment by praveenrewar** [comment]
Fixed with #1556
